pkgs/by-name/go/google-chrome/package.nix

```diff
@@ -159,116 +168,198 @@ let
       gtk4
     ];
 
-in
-stdenv.mkDerivation (finalAttrs: {
-  pname = "google-chrome";
-  version = "127.0.6533.88";
+  linux = stdenv.mkDerivation (finalAttrs: {
+    inherit pname meta;
+    version = "127.0.6533.88";
 
-  src = fetchurl {
-    url = "https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_${finalAttrs.version}-1_amd64.deb";
-    hash = "sha256-0l9cidNFO0dcyzWy4nDD/OGFQDBLXx9aPVq6ioDkqK0=";
-  };
+    src = fetchurl {
+      url = "https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_${finalAttrs.version}-1_amd64.deb";
+      hash = "sha256-0l9cidNFO0dcyzWy4nDD/OGFQDBLXx9aPVq6ioDkqK0=";
+    };
+
```

Comment:
> strictDeps = true;

It's a good habit to keep `strictDeps` on, to help separate what belongs to `nativeBuildInputs` and `buildInputs`.